README.md

```diff
@@ -0,0 +1,42 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Coffee and Code Annual meeting
+
+> Coffee and Code helds annual meeting every year. This website gives detail information on the annual meeting. Users all over the world can see who are the choosen featured speakers and detail infromation about previous events.
+
```

Comment:
- [ ] **[OPTIONAL]** I think it would be nice to include the screenshot of your app in the README

Reply:
@TedLivist  Thanks I will do it on the main branch